app/src/main/kotlin/com/wire/android/ui/userprofile/teammigration/TeamMigrationViewModel.kt

```diff
@@ -17,18 +17,25 @@
  */
 package com.wire.android.ui.userprofile.teammigration
 
+import androidx.compose.foundation.text.input.setTextAndSelectAll
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.setValue
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
 import com.wire.android.feature.analytics.AnonymousAnalyticsManager
 import com.wire.android.feature.analytics.model.AnalyticsEvent
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.feature.user.migration.MigrateFromPersonalToTeamResult
+import com.wire.kalium.logic.feature.user.migration.MigrateFromPersonalToTeamUseCase
 import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
 class TeamMigrationViewModel @Inject constructor(
-    private val anonymousAnalyticsManager: AnonymousAnalyticsManager
+    private val anonymousAnalyticsManager: AnonymousAnalyticsManager,
+    private val teamMigrationUseCase: MigrateFromPersonalToTeamUseCase
```

Comment:
I'd suggest to use action oriented names as it's more clear.
`teamMigrationUseCase` does not provide an immediate understanding of what the use case is actually doing.
```suggestion
    private val migrateFromPersonalToTeam: MigrateFromPersonalToTeamUseCase
```